_data/developer.yml

```diff
@@ -224,6 +230,7 @@ websites:
     - name: Papertrail
       url: https://papertrailapp.com/
       twitter: papertrailapp
+      facebook: papertrail.io
```

Comment:
I concur. The Facebook page for [papertrailapp.com](https://papertrailapp.com/) appear to be @[papertrailapp](https://www.facebook.com/papertrailapp/), but it is not verified and has virtually no activity.